samples/hello_events.c

```diff
@@ -174,7 +175,9 @@ int main(int argc, char *argv[])
 			 goto out_destroy_eh;
 		} else {
 			 printf("FME Interrupt occured\n");
-			 read(pfd.fd, &count, sizeof(count));
+			 bytes_read = read(pfd.fd, &count, sizeof(count));
+			 if (bytes_read == 0)
```

Comment:
We should probably handle the case where there is an error and bytes_read is negative.  `man 3 read` says:

````
RETURN VALUE
       Upon successful completion, these functions shall return a non-negative
       integer indicating the number of bytes actually  read.  Otherwise,  the
       functions shall return −1 and set errno to indicate the error.
````

Reply:
Since this PR was merge, I'll put that in a new PR